### DISCUSSION TITLE
TTS Beginner Questions

### DISCUSSION THREAD

**Discussion by Suvanch** [coqui-ai/TTS#1202] [General Q&A]
Hello,

I am new to CS and I thought this project was really cool. I wanted to try this out for myself, however, I am having trouble understanding the documentation. So here are some of the questions I had

Can Corqui-ai TTS do real-time voice cloning like CorentinJ/Real-Time-Voice-Cloning? 
            -CorentinJ RTVC recommended this page for voice cloning. CorentinJ RTVC had ok quality for high sp. but I'm curious to see 
             how much better it can get.
If I can do real-time voice cloning, how do I try it out?

I also want to try to fully clone my voice so how can I train my own model?
-What model should I train it with for the best quality?
-I also don't have a GPU will I be still able to train my own model with my CPU? (I have an Intel Corei7 8th Gen) 
-How long would this process take and will I be able to use my computer while it's training the model?
-How do I train my own model/ Where can I find the instructions for training?

I have followed the instructions on the read me and got some results however I want to hear some different voices.
How do I know which models have multispeaker models? because when I run
           tts --model_name "<language>/<dataset>/<model_name>"  --list_speaker_idxs with most models I get an error saying model 
           has no no attribute 'speaker_ids'

What is the difference between an encoder, vocoder, and synthesizer?

Thank you for being patient with me and helping me out. :)

**Comment by Ca-ressemble-a-du-fake** [comment]
Hi @Suvanch, regarding your question on voice cloning, you can try the demo on the coqui homepage and input 1 minute of your voice. The tts output should sound as your voice.

Regarding your concern on training with CPU, I don't think it will be fast enough. But you can use Google Colab and use a GPU for free. There is a voice conversion Colab notebook available that will do what you're seeking.
